cpp/src/cider/function/scalar/RuntimeFunctions.h

```diff
@@ -23,14 +23,16 @@
 #ifndef CIDER_FUNCTION_RUNTIMEFUNCTIONS_H
 #define CIDER_FUNCTION_RUNTIMEFUNCTIONS_H
 
-#include "type/data/funcannotations.h"
 
 #include <cassert>
 #include <cstdint>
 #include <ctime>
 #include <limits>
 #include <type_traits>
 
+#include "exec/nextgen/function/CiderStringFunction.h"
```

Comment:
maybe the runtime function should move out of the nextgen directory.
and put them into `cider/function/` directly.